iOS Client/Store/XMLParsingDelegate.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  XMLParsingDelegate.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 12/9/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import Foundation
+
+
+class PodcastXMLParsingDelegate: NSObject, XMLParserDelegate {
+    var podcast: [String: String] = [:]
+    var items: [[String:String]] = []
+    var podcasts: [MBPodcast] = []
+    var foundCharacters: String = ""
+    
+    func parser(_ parser: XMLParser, foundCharacters string: String) {
+        self.foundCharacters += string
+    }
+    
+    func parser(_ parser: XMLParser, didEndElement elementName: String, namespaceURI: String?, qualifiedName qName: String?) {
+        if elementName == "title" {
+            self.podcast["title"] = self.foundCharacters.trimmingCharacters(in: .whitespacesAndNewlines)
+        } else if elementName == "author" {
+            self.podcast["itunes:author"] = self.foundCharacters.trimmingCharacters(in: .whitespacesAndNewlines)
+        } else if elementName == "description" {
+            self.podcast["description"] = self.foundCharacters.trimmingCharacters(in: .whitespacesAndNewlines)
+        } else if elementName == "itunes:image" {
```

Comment:
do any of these have images?

Reply:
I think some of them return images, I don't do anything with them at the moment tho